### PR TITLE
Update RecordService import

### LINKED ISSUE #86: RecordService type import should be outside brackets
**Describe the bug**
My typescript language server is giving me errors on the generated file, specifically when we import the RecordService
image

A solution would be moving `type` outside the brackets
image

**Versions**
pocketbase-typegen version: 1.2.0 
pocketbase version: 0.18.9

### PR DESCRIPTION
**Pull request by seofernando25**
Fixes #86

### CODE REVIEW

**Comment by seofernando25** [comment]
Oops need to update pkg ver

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
   "author": "@patmood",
   "license": "ISC",
   "dependencies": {
+    "ci": "^2.3.0",
```
Comment (patmood) [inline package.json:36]: I think this is breaking the github actions. Is it necessary?

**Review by patmood — COMMENTED** [review]
Looks good, just need to remove the `ci` dependency I think

File: package.json (hunk 1 above)
Comment (seofernando25) [reply package.json]: Uh oh... sorry about that, I don't remember making this change.

I think I ran `npm i ci` by accident instead of `npm ci`

**Comment by patmood** [comment]
Just need to update a couple of snapshots with `npm run test:update`. By running the test suite it'll also generate a new example output, which should make the integration tests pass too 👍

**Comment by seofernando25** [comment]
Oh you have snapshots! Cool I wasn't aware of that.

**Comment by patmood** [comment]
Thanks for the contribution!

**Pull request merged**